scanner/src/main/kotlin/storages/utils/ScanResultDao.kt

```diff
@@ -30,7 +30,7 @@ import org.ossreviewtoolkit.model.ScanResult
 
 object ScanResults : IntIdTable("scan_results") {
     val identifier: Column<String> = text("identifier").index("identifier")
-    val scanResult: Column<ScanResult> = jsonb("scan_result", ScanResult::class)
+    val scanResult: Column<ScanResult> = jsonb("scan_result")
```

Comment:
See above. As a side note, line 32 could also omit the explicit type, if desired.